package.json

```
@@ -13,6 +13,6 @@
     "test:withdraw": "node index.js --cmd=withdraw --validate"
   },
   "dependencies": {
-    "kraken-api": "1.0.0"
+    "kraken-api": "^1.0.1"
```

```suggestion
    "kraken-api": "1.0.1"
```

Please use a fixed version here – I wouldn't want this to change unnoticedly.

---

Fixed. 

Still got a question: The version of the dependencies are fixed in the package-lock.json and the content will change if packages are added / updated, so wouldn't changes be noticed there?

---

Thanks! The way it is right now leads to no unintended changes being installed, since the [package-lock.json specifies everything](https://medium.com/coinmonks/everything-you-wanted-to-know-about-package-lock-json-b81911aa8ab8). By specifying `"kraken-api": "^1.0.1"` as dependency we'd say "I'll take 1.0.1, but I'd also be fine with 1.0.9", which we should prevent, because the user might not notice the changes when running `npm install`.